frontend/app/dashboard/fit_score_chart.tsx

```diff
@@ -0,0 +1,55 @@
+import { Card, CardContent, Rating, Typography } from "@mui/material";
+import "./dashboard.css";
```

Comment:
please use css modules like we are using in the rest of the project

look at app/counter.tsx for an example.